kbld.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: kbld
+  version: 0.44.1
+  epoch: 0
+  description: kbld seamlessly incorporates image building and image pushing into your development and deployment workflows
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: fac5bb8f504df2d5231aeb383418112c592ca15c
+      repository: https://github.com/carvel-dev/kbld
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      packages: ./cmd/kbld
+      output: kbld
+      ldflags: -X carvel.dev/kbld/pkg/kbld/version.Version=v${{package.version}}
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: carvel-dev/kbld
+    strip-prefix: v
+
+test:
+  pipeline:
+    - name: version check
+      runs: |
+        kbld version | grep ${{package.version}}
```

Comment:
Is it possible to run simple example on kwok using `kblf -f` as in the [example](https://github.com/carvel-dev/simple-app-on-kubernetes#step-3-building-container-images-locally) without need having `kapp` dep? (or we should merge it first)